packages/components/src/Form/tests/chromatic/Form.stories.tsx

```diff
@@ -0,0 +1,200 @@
+                    <Label>Adress</Label>
+                </TextField>
+                <TextField placeholder="Apt or suite">
+                    <Label>Apt/Suite</Label>
+                </TextField>
+            </Inline>
+            <TextField placeholder="Old El Paso">
+                <Label>City</Label>
+            </TextField>
+            <TextArea placeholder="Leave a comment">
+                <Label>Comments:</Label>
+            </TextArea>
+            <Checkbox>
+                Agree to terms and conditions
+            </Checkbox>
+            <Button>Submit</Button>
+        </>);
+};
+
+export const Default = {
+    args: {
+        children: <Template />
+    }
+} satisfies Story;
+
+export const Small = {
+    args: {
+        ...Default.args,
+        size: "sm"
+    }
+};
+
+export const Disabled = {
+    args: {
+        ...Default.args,
+        isDisabled: true
+    }
+};
+
+export const Fluid = {
+    args: {
+        ...Default.args,
+        isFluid: true
+    }
+};
+
+export const Validation = {
+    render: () => (
+        <Form validationErrors={{ username: "Sorry, this username is taken." }}>
+            <TextField defaultValue="john_doe" name="username">
+                <Label>Username</Label>
+                <ErrorMessage />
+            </TextField>
+        </Form>
+    )
+};
+
+export const ValidationBehaviorAria = {
+    render: () => (
+        <Form validationBehavior="aria">
+            <TextField
+                name="username"
+                defaultValue="admin"
+                isRequired
+                validate={value => value === "admin" ? "Nice try." : null}
+            >
+                <Label>Username</Label>
+                <ErrorMessage />
+            </TextField>
+            <Button type="submit">Submit</Button>
+        </Form>
+    )
+};
+
+export const ValidationBehaviorNative = {
+    render: () => {
+        return (
+            <Form validationBehavior="native">
+                <TextField
+                    name="email"
+                    type="email"
+                    isRequired
+                >
+                    <Label>Email</Label>
+                    <ErrorMessage>This field is required</ErrorMessage>
+                </TextField>
+                <ButtonGroup>
+                    <Button type="submit">Submit</Button>
+                    <Button type="reset">Reset</Button>
+                </ButtonGroup>
+            </Form>
+        );
+    },
+    play: async ({ canvasElement }: { canvasElement: HTMLElement }) => {
+        const canvas = within(canvasElement);
+        const [submitButton] = canvas.getAllByRole("button");
+
+        await userEvent.click(submitButton);
+
+        const errorRender = await canvas.findByText("This field is required");
+        await expect(errorRender).toBeVisible();
+    }
+};
+
+export const Styling = {
+    parameters: {
+        ...a11yParameters({ disableContrastCheck: true })
```

Comment:
instead of disabling, should we just choose a different style that doesn't cause errors?